solutions/prob6.py

```diff
@@ -0,0 +1,34 @@
+# https://projecteuler.net/problem=6
+# Sum square difference of first 100 natural numbers
+
+import math
+
+class sumSquareDiff(object):
+	
+	def __init__(self, number):
+        	self.number = number
```

Comment:
Could you please follow [PEP8](https://www.python.org/dev/peps/pep-0008/) rules? 
You will need to modify indentation, naming convention of class, variables, add spaces.